napari/_qt/qt_main_window.py

```diff
@@ -158,7 +158,9 @@ def __init__(
 
         self.status_throttler = QSignalThrottler(parent=self)
         self.status_throttler.setTimeout(50)
+        self._reconnect_cursor(viewer)
 
+    def _reconnect_cursor(self, viewer: 'Viewer'):
```

Comment:
Optional: consider a rename to clarify the goal of this helper method - e.g. `_throttle_cursor_to_status_connection`.

Reply:
changed